source/features/hide-zero-packages.tsx

```diff
@@ -2,16 +2,20 @@ import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import getTabCount from './remove-projects-tab';
 
-async function init(): Promise<void> {
+async function init(): Promise<void | false> {
 	const packagesCounter = await elementReady([
-		'.numbers-summary [href$="/packages"] .num', // `isRepoRoot`
-		'.UnderlineNav-item[href$="?tab=packages"] .Counter' // `isUserProfile`
-	].join(','));
+		'.numbers-summary [href$="/packages"]', // `isRepoRoot`
+		'.UnderlineNav-item[href$="?tab=packages"]', // `isUserProfile`
+		'.orgnav > .pagehead-tabs-item[href$="/packages"]:not(.selected)' // `isOrganizationProfile`
```

Comment:
Shouldn't all of them have `:not(.selected)`?

Reply:
Correct the `isUserProfile` since `isRepoRoot` is not a tab